benchmarks/benchmarks.py

```diff
@@ -20,20 +20,33 @@
 
 import networkx as nx
 import numpy as np
+from typing import Callable
 
 import cirq
 from mitiq import benchmarks, raw, pec, zne, Observable, PauliString
 from mitiq.interface import mitiq_cirq
+from mitiq.zne.scaling import fold_gates_at_random, fold_global
 
 
 if os.environ.get("BENCHMARK_CI"):
-    params = {"nqubits": [1], "depth": [1, 2, 3], "num_pec_samples": [10]}
+    params = {
+        "nqubits": [1],
+        "depth": [1, 2, 3],
+        "num_pec_samples": [10],
+        "fold_method": [fold_gates_at_random],
+    }
 else:
-    params = {"nqubits": [2], "depth": [1, 5, 10], "num_pec_samples": [100]}
+    params = {
+        "nqubits": [2],
+        "depth": [1, 5, 10],
+        "num_pec_samples": [100],
```

Comment:
```suggestion
        "num_pec_samples": [100, 500],
```
If you think this is too crazy in terms of execution time, I would propose  "num_pec_samples": [50, 100].
Just to have a rough idea of how the result depends on the number of PEC samples

Reply:
500 slowed things down quite a bit. Did 50, and 100 instead in https://github.com/unitaryfund/mitiq/pull/1374/commits/3e595cd213315f3c72f972522a5019a75512cab7